src/utils/user/email-verification.ts

```diff
@@ -0,0 +1,138 @@
+import { ENV } from '../env';
+import { generateTicketExpiresAt } from '../ticket';
+import { v4 as uuidv4 } from 'uuid';
+import { insertUser } from './insert';
+import { getGravatarUrl } from '../avatar';
+import { EMAIL_TYPES, UserRegistrationOptions } from '@/types';
+import { hashPassword } from '../password';
+import { emailClient } from '@/email';
+import { createEmailRedirectionLink } from '../redirect';
+import { getUserByEmail } from './getters';
+
+const sendEmailIfNotVerified = async (
+  email: string,
+  newEmail: string,
+  user: any,
+  displayName: string,
+  ticket: string,
+  redirectTo: string
+) => {
```

Comment:
We can improve the readability and maintenance in wrapping params in an object:

```suggestion
import { UserQuery } from '../__generated__/graphql-request';

const sendEmailIfNotVerified = async ({
  email,
  newEmail,
  user,
  displayName,
  ticket,
  redirectTo,
}: {
  email: string;
  newEmail: string;
  user: NonNullable<UserQuery['user']>;
  displayName: string;
  ticket: string;
  redirectTo: string;
}) => {```